pp/integrations/plugin-packs/procedures/operatingsystems-freebsd-snmp.md

```diff
@@ -314,154 +312,154 @@ yum install centreon-plugin-Operatingsystems-Freebsd-Snmp
-|:-------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------|:-----------:|
-| FILTER       | Set the storage name, it can be a regex (empty means 'check all storage')                                                                                                                      | .*                                                      |             |
-| TRANSFORMSRC | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run' will replace all occurrences of 'dev' with 'run' |                                                         |             |
-| TRANSFORMDST | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run' will replace all occurrences of 'dev' with 'run' |                                                         |             |
-| WARNING      | Warning threshold                                                                                                                                                                              | 80                                                      |             |
-| CRITICAL     | Critical threshold                                                                                                                                                                             | 90                                                      |             |
-| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                         | --verbose --filter-perfdata='storage.space\|used\|free' |             |
+| Macro        | Description                                                                                                                                                                                     | Default value                                           | Mandatory   |
+|:-------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------|:-----------:|
+| FILTER       | Set the storage (number expected) example: 1, 2,... (empty means 'check all storage')                                                                                                           | .*                                                      |             |
```

Comment:
```suggestion
| FILTER       | Set the storage name, it can be a regex (empty means 'check all storage')                                                                                                           | .*                                                      |             |
```